Lvl99GameJam/Assets/Scripts/UI Scripts/GridSpaceInteractions.cs

```diff
@@ -96,11 +128,6 @@ private void Update()
             if (Input.GetKeyDown(keyCode) && placementPrefab != prefab)
             {
                 placementPrefab = prefab;
-                if (containsPointer)
```

Comment:
Want to still change preview sprite when 1 or 2 is pressed, right? If not, can remove `containsPointer` I think.